### PR TITLE
Add linkerd proposal

### PR DESCRIPTION
**Pull request by caniszczyk**
https://linkerd.io/

Signed-off-by: Chris Aniszczyk <caniszczyk@gmail.com>

### CODE REVIEW

**Comment by caniszczyk** [comment]
RFC @cncf/toc 

Also cc: @olix0r @wmorgan @jonboulle

**Comment by caniszczyk** [comment]
One thing to note is that since the graduation criteria recently became solidified (https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc), the intention is to have linkerd come in as an "inception" stage project.

**Comment by duglin** [comment]
@caniszczyk should the application have a field asking for level each project is asking for?

**Comment by caniszczyk** [comment]
@duglin yes, created the issue here to track https://github.com/cncf/toc/issues/26

**Comment by duglin** [comment]
@caniszczyk cool - thanks

File: proposals/linkerd.adoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Applications use linkerd by proxying HTTP, gRPC, or Thrift calls through a local linkerd instance. In many cases, integration with linkerd is a config change (e.g. setting an http_proxy environment variable) rather than a code change. Linkerd acts as a transparent proxy, and so a call to (e.g. "http://foo/bar") will be routed to the appropriate cluster, resolved in service discovery, load balanced, retried, and returned without application code being aware.
+
+Linkerd is built on top of Finagle (https://twitter.github.io/finagle/), the high-concurrency RPC library that powers companies like Twitter, Soundcloud, Pinterest, and ING Bank. By staying close to the Finagle codebase, linkerd takes advantage of Finagle's extensive production usage (https://github.com/twitter/finagle/blob/develop/ADOPTERS.md). Linkerd’s primary author is a core contributor to Finagle.
+
+*Sponsor / Advisor from TOC*: Jonathan Boulle <jonathan.boulle@coreos.com>
+
+*Unique Identifier*: linkerkd
```
Comment (wmorgan) [inline proposals/linkerd.adoc]: linkerd

**Comment by caniszczyk** [comment]
We plan on voting on this proposal early next week on the TOC list.

We also plan to consider this project as "inception" stage and will update the proposal accordingly.

**Comment by caniszczyk** [comment]
Final RFC from @cncf/toc and the wider community, I plan on start the official voting tomorrow.

cc: @monadic

File: proposals/linkerd.adoc (hunk 1 above)
Comment (caniszczyk) [inline proposals/linkerd.adoc]: fixed, thanks

**Pull request merged**